gometalinter.json

```diff
@@ -19,5 +19,6 @@
     "ineffassign",
     "misspell",
     "vet"
-  ]
+  ],
+  "Exclude": ["presidio-api/cmd/presidio-api/docs"]
```

Comment:
is this needed?
did we get docs lint? so why not keep it?

Reply:
The linter fails on unused objects. 
In these files we create structures as wrappers containing parameters, to be referenced by go-swagger when generating the swagger file. These structures are not used by any go functions, so the linter thinks they are "un-usable".